### PR TITLE
MSSQL Metrics (Part II).

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This PR brings final metrics we were delivering using `go.d.plugin`. 

##### Test Plan

1. Get MSSQL for developers and install it on your computer. You can get it using [this](https://www.microsoft.com/en-us/sql-server/sql-server-downloads) link.
2. Compile this Branch
3. Create the installer.
4. Install and take a look in the metrics.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: src/collectors/windows.plugin/perflib-mssql.c [hunk 1]
```diff
@@ -730,17 +791,667 @@ static void do_mssql_errors(PERF_DATA_BLOCK *pDataBlock, struct mssql_instance *
...
+                                                        , RRDSET_TYPE_LINE
+                                                        );
+
+            snprintfz(id, RRD_ID_LENGTH_MAX, "mssql_instance_%s_memmgr_connection_memory_bytes", p->instanceID);
+            p->rd_conn_memory  = rrddim_add(p->st_conn_memory,
+                                           id,
+                                           "memory",
+                                           1,
+                                           1,
+                                           RRD_ALGORITHM_INCREMENTAL);
```
Comment (stelfrag) [inline src/collectors/windows.plugin/perflib-mssql.c:1332]: Is this incremental?

File: src/collectors/windows.plugin/perflib-mssql.c [hunk 2]
```diff
@@ -730,17 +791,667 @@ static void do_mssql_errors(PERF_DATA_BLOCK *pDataBlock, struct mssql_instance *
...
+                                                           , RRDSET_TYPE_LINE
+                                                           );
+
+            snprintfz(id, RRD_ID_LENGTH_MAX, "mssql_instance_%s_memmgr_total_server_memory_bytes", p->instanceID);
+            p->rd_mem_tot_server  = rrddim_add(p->st_mem_tot_server,
+                                              id,
+                                              "memory",
+                                              1,
+                                              1,
+                                              RRD_ALGORITHM_INCREMENTAL);
```
Comment (stelfrag) [inline src/collectors/windows.plugin/perflib-mssql.c:1431]: Is this incremental? Please check the others as well

File: src/collectors/windows.plugin/perflib-mssql.c (hunk 1 above)
Comment (thiagoftsm) [reply src/collectors/windows.plugin/perflib-mssql.c]: Thank you @stelfrag. We were using `incremental` in our `go.d.plugin`, but they are gauge.

File: src/collectors/windows.plugin/perflib-mssql.c (hunk 2 above)
Comment (thiagoftsm) [reply src/collectors/windows.plugin/perflib-mssql.c]: I also checked others and I push a commit to address them. Thanks!

**Review by stelfrag — APPROVED** [review]

**Pull request merged**